modules/core/src/v2/wallet.ts

```diff
@@ -1625,9 +1625,16 @@ export class Wallet {
     const self = this;
     return co<PrebuildTransactionResult>(function *() {
       // Whitelist params to build tx
-      const whitelistedParams = _.pick(params, self.prebuildWhitelistedParams());
+      let whitelistedParams = _.pick(params, self.prebuildWhitelistedParams());
```

Comment:
should the default be set in `prebuildWhitelistedParams()` perhaps? we leak coin-specific code here that could be avoided

Reply:
It's a good idea, but I think this just returns the property names to take from the params object and can't actually set any defaults. Would be good to not have coin specific code in here, but it would require a bit of refactoring.

Reply:
ah yes of course

weird that we made it this far without having a func providing coin-specific defaults here

let's not refactor in this PR imo